plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -61,9 +61,6 @@
         id="com.google.cloud.tools.eclipse"
         name="Google Cloud Platform"
         class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
-       <enabledWhen>
-         <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
This will now show the Google Cloud Platform root preference on any project (e.g., plain Java project). I'm not sure if we want that.

(BTW, if I'm not mistaken, the name `onlyInGCPProjects` is misleading in that it actually means `onlyInProjectsWithAppEngineStandardFacet` at the moment.)